spaceship/lib/spaceship/tunes/tunes_client.rb

```diff
@@ -971,6 +972,18 @@ def send_app_submission(app_id, version, data)
 
       handle_itc_response(r.body)
 
+      messages = r.body.fetch("messages", {})
+      errors = messages.fetch('error', nil)
+
+      if !errors.nil? and !errors.empty? and errors.first.include?("Problem processing review submission.")
```

Comment:
I think you could use the Ruby safe navigation operator here to avoid the explicit nil checks:

```ruby
if errors&.first&.include?("Problem processing review submission.")
```

Reply:
Wow, TIL: safe nav operator.
That's a great suggestion, but unfortunately we're tied to Ruby 2.0.0 😢 and it looks like the safe nav operator was introduced in 2.3.0

Reply:
based on @taquitos's comment, keeping the existing code. Not making requested change.

Reply:
But I think we could use `Array(errors).include?("something")`, right?